### PR TITLE
Users can chat with other online users.

### PR DESCRIPTION
**Pull request by danielduffield**
Created a chat sidebar that allows users to chat with each other, either with a guest id or a chosen nickname.

### CODE REVIEW

File: client/utils/canvas.js [hunk 1]
```diff
@@ -158,17 +160,41 @@ export default class Canvas extends React.Component {
     this.painting = false
   }
   render() {
+    this.isChatHidden = this.props.isChatHidden
+    const SecondWrapper = styled.div`
```
Comment (thebearingedge) [inline client/utils/canvas.js]: It's best to move these styled component definitions outside of your render methods.

**Review by thebearingedge — APPROVED** [review]

**Comment by danielduffield** [comment]
https://con-canvas.herokuapp.com/

**Comment by danielduffield** [comment]
concanvas-issue-9

**Review by thebearingedge — APPROVED** [review]
Consider accessing `dispatch` on props in connected components.

**Pull request merged**